apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/common/ResourceTypeV2.java

```diff
@@ -13,6 +13,9 @@
  */
 public interface ResourceTypeV2<TResource extends IBaseResource, TEntity> {
 
+  /** @return A label for the resource type that uniquely identifies it */
+  String getTypeLabel();
+
   /** @return a name for use when building drop wizard metric names */
   String getNameForMetrics();
```

Comment:
Why retain this?

Reply:
As you would probably guess, I was thinking `getNameForMetrics()` wasn't intuitive for my use, but I was on the fence about changing the method name for the existing logic that uses that one.  I guess it's probably fine to just change it though.

Reply:
Yeah I think getTypeLabel is fine for both.